doc/base/classes.xml

```diff
@@ -2007,32 +2007,49 @@
 </class>
 <class name="ARVRAnchor" inherits="Spatial" category="Core">
 	<brief_description>
+		Anchor point in AR Space
 	</brief_description>
 	<description>
+		The ARVR Anchor point is a spatial node that maps a real world location identified by the AR platform to 
```

Comment:
One line should be one paragraph if I'm not mistaken.

Reply:
Ah!